docs/core-concepts/data ingestion/why-ingestion.mdx

```diff
@@ -35,68 +22,214 @@ One solution to this is to build data around **Cloud Optimized formats**: Data l
+    with open("data.zip", "wb") as f:
+        f.write(response.content)
+    
+    with zipfile.ZipFile("data.zip", "r") as zip_ref:
+        zip_ref.extractall("./data")
 
-:::warning
-    Fused does not (yet) have a build-in tool to ingest raster data. We suggest you create COGs yourself, for example by using `gdal`'s [built-in options](https://gdal.org/en/stable/drivers/raster/cog.html#examples) or [`cogger`](https://github.com/airbusgeo/cogger?tab=readme-ov-file#usage)
-    {/* TODO: Link to how to run CLI commands in Fused once we have this up and running */}
-:::
+    csv_df = fused.api.upload("./data/AIS_2024_01_01.csv", "fd://demo_reading_ais/AIS_2024_01_01.csv")
+    print(f"Saved data to fd://demo_reading_ais/")
 
-Cloud Optimized GeoTiffs have multiple different features making them particularly interesting for cloud native applications, namely:
-- **Tiling**: Images are split into smaller tiles that can be individually accessed, making getting only parts of data a lot faster.
-- **Overviews**: Pre-rendered images of lower zoom levels of images. This makes displaying images at different zoom levels a lot faster
+    return pd.DataFrame({"status": 'Done'})
+```
 
-import ImgCOGE84 from '@site/docs/core-concepts/data ingestion/e84_cog_smiley_tiled.png';
+And simply running this UDF:
+
+```python
+fused.run(save_ais_csv_to_fused_udf)
+```
+
+We can check that our CSV was properly ingested with File Explorer by navigating to `fd://demo_reading_ais/`:
+
+import ImgFileExplVerification from '@site/docs/core-concepts/data ingestion/FileExplorer_Demo_AIS.png';
 
 <div style={{textAlign: 'center'}}>
-<img src={ImgCOGE84} alt="A simple overview of Geoparquet benefits" style={{width: 400}} />
+<img src={ImgFileExplVerification} alt="Our AIS CSV properly uploaded on File Explorer" style={{width: 600}} />
 </div>
-_A simple visual of COG tiling: If we only need the top left part of the image we can fetch only those tiles (green arrows). Image courtesy of [Element 84's blog on COGs](https://element84.com/software-engineering/remote-sensing/cloud-optimized-geotiff-vs-the-meta-raster-format)_
 
-- Element84 wrote a [simple explainer of what Cloud Optimized GeoTiffs are](https://element84.com/software-engineering/remote-sensing/cloud-optimized-geotiff-vs-the-meta-raster-format) with great visuals
-- [Cloud Optimized Geotiff spec dedicated website](https://cogeo.org/)
-- [Cloud Optimized Geotiff page on Cloud Native Geo guide](https://guide.cloudnativegeo.org/cloud-optimized-geotiffs/intro.html)
+That's one big CSV.
+
+But opening it on its own doesn't do all that much for us. We're going to create 3 [UDFs](/core-concepts/write/) to showcase a more real-world application: Opening the dataset and returning a subset inside a bounding box. We'll do this 3 different ways to compare their execution time:
+- 1. From the default CSV
+- 2. From the same data but saved a `.parquet`
```

Comment:
```suggestion
- 2. From the same data but saved as a `.parquet`
```

Reply:
BTW, did you push for those? (you marked it as resolved but it does not actually seem changed; unless you disagreed with this suggestion of course ;))